Resources/Private/Javascript/Connector.js

```diff
@@ -42,71 +60,46 @@ export class ShopwareConnector {
     }
   }
 
-  addProductToCart(productId) {
-    const url = endpoints.ADD_TO_CART_BASE + productId;
+  request(method, url, parameter) {
```

Comment:
not sure if im happy with this change, i see its advantages but it might also be a a set of handcuffs if you try to press everything in the same format...i would keep it for now but rename it to _request (marking it as a somewhat private method) and switch to named parameter pattern e.g. https://medium.com/@afontcu/cool-javascript-9-named-arguments-functions-that-get-and-return-objects-337b6f8cfa07